webapp/webpack.config.ts

```diff
@@ -55,6 +55,11 @@ module.exports = {
                 exclude: /node_modules/,
                 use: 'ts-loader',
             },
+            {
+                test: /\.svg$/,
+                issuer: /\.[jt]sx?$/,
+                use: ['@svgr/webpack', 'url-loader'],
```

Comment:
This is removing the viewbox from the svgs in the plugin. 
See the screenshot - 
![image](https://github.com/Brightscout/mattermost-plugin-msteams-sync/assets/77336594/abe01b5d-7efe-453d-bae5-d924ffe37a48)

The viewbox is present in the mm-ui-library but is removed here.
![image](https://github.com/Brightscout/mattermost-plugin-msteams-sync/assets/77336594/ca47617e-23c1-4a5f-ad19-b8644057b0b4)